content/id/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -0,0 +1,269 @@
+---
+title: DNS untuk Service dan Pod
+content_template: templates/concept
+weight: 20
+---
+{{% capture overview %}}
+Laman ini menyediakan ikhtisar dari dukungan DNS oleh Kubernetes.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Pendahuluan
+
+Kubernetes DNS melakukan _scheduling_ DNS Pod dan Service yang ada pada kluster, serta 
+melakukan konfigurasi kubelet untuk memberikan informasi bagi setiap container 
+untuk menggunakan DNS Service IP untuk melakukan resolusi DNS.
+
+### Apa Sajakah yang Mendapatkan Nama DNS?
+
+Setiap Service yang didefinisikan di dalam kluster (termasuk server DNS itu sendiri) 
+memiliki nama DNS. Secara default, sebuah list pencarian DNS pada Pod klien
+akan mencantumkan namespace Pod itu sendiri serta domain default kluster. Hal ini dapat diilustrasikan 
+dengan contoh berikut:
+
+Asumsikan sebuah Service dengan nama `foo` pada Kubernetes dengan namespace `bar`. 
+Sebuah Pod yang dijalankan di namespace `bar` dapat melakukan resolusi 
+terhadap service ini dengan melakukan _query_ DNS 
+untuk `foo`.  Sebuah Pod yang dijalankan pada namespace `quux` dapat melakukan 
+resolusi service ini dengan melakukan _query_ DNS untuk `foo.bar`.
+
+Bagian di bawah ini akan menampilkan detail tipe rekord serta _layout_ yang didukung. 
```

Comment:
rekord is not Indonesian word, how about `rekaman` or simply _record_ to align with A record and other DNS record terminology?
```suggestion
Bagian di bawah ini akan menampilkan detail tipe rekaman serta _layout_ yang didukung. 
```